bundles/net.tourbook.common/src/net/tourbook/common/time/TimeTools.java

```diff
@@ -343,6 +327,24 @@ public static TourDateTime createTourDateTime(final long epochMilli, final Strin
       return new TourDateTime(tourZonedDateTime, timeZoneOffsetLabel, weekDays_Short[weekDayIndex]);
    }
 
+   /**
+    * @return Returns a long value for the given date/time in the YYYYMMDDhhmmss format
+    */
+   public static long createYMDhms_From_DateTime(final ZonedDateTime zonedDateTime) {
+
+      final long dtYMDhms = 0
+
+            + (zonedDateTime.getYear() * 10000000000L)
```

Comment:
Long values are easier to read with `_` 

`10000000000L` -> `10_000_000_000L`

Reply:
Fixed in several places